docs/templates/snakemake/config/config.[METHOD].yaml

```diff
@@ -11,6 +11,27 @@ method: "METHOD" # same as the name of the directory
 
 
 # Parameters
+
+# Flags to run/output files for each benchmarking challenge
+# REMOVE the flag from the file if the tool does not output files compatible with a challenge
+# All flags are 'True/False' to 'switch on/off' rules to produce challenge output. Valid parameter values are:
+# 'True': Run/output challenge files
+# 'False': Don't run/output challenge files
+# Note: Do not enclose True/False in quotation marks
+# If combinations of flags are incompatible with one another (e.g. run_differential requires a different workflow to run_quantification), this should be noted here and in the README
+
+
+# Whether to run steps to produce output files for identification challenge
+run_identification: True
+
+run_quantification: True
+
+run_differential: True
+
+identification_output_suffix: ".identification.bed"
+quantification_output_suffix: ".quantification.bed"
+differential_output_file: "differential_challenge.tsv"
```

Comment:
Hmm, There wasn't much thought in the specific values other than to get something dry-runnable. I chose output suffixes for ID & quant mainly because the Ewfs and run specifications I've seen so far are per-sample, so just wanted a way to differentiate between the different challenge files. I guess this is based on the assumption all challenge output files will go to the same output directory, but that may not actually reflect our documentation/how we plan to run each challenge. 

I now realise I should be very careful here as I don't want to introduce conflicts between the documentation and the template. Judging from [output filenames specifications](https://github.com/iRNA-COSI/APAeval/tree/main/execution_workflows#filenames), we want our output filenames to be `CHALLENGECODE_PARTICIPANT_OUTCODE.ext`, where:

- CHALLENGECODE - in my head analogous to a sample ID, in which case defined in `samples.csv`?
- PARTICIPANT - name of tool/workflow (need to add this to the config)
- OUTCODE - [01 for ID, 02 for quantification & 03 for differential](https://github.com/iRNA-COSI/APAeval/blob/main/execution_workflows/execution_output_specification.md#outputs)
 
so I need to update the template & `config.yaml` to reflect this.

With that in mind I should also update the `*_output_suffix` parameters to be:
- "{challenge code}.bed" for ID & quant, where challenge codes are 01 & 02 respectively
- "03.tsv" for differential challenge (replacing `differential_output_file` with `differential_output_suffix`)

I could also have the challenge codes and extensions as separate parameters if we want more customisation, but ultimately they'd be joined together anyway. In my head 'challenge code' is analogous to a sample ID, in which case it would be defined in `samples.csv`, but that may not be the case for all of our challenges.

Reply:
Thanks for the explanations :) 
I think the current state is good, because it is flexible and easy to adjust. So I don't think we need to do something here, just to keep in mind how we want to save the runs in the end.